antlr4ide.tests/src/main/java/com/github/jknack/antlr4ide/generator/ToolOptionsTest.java

```diff
@@ -136,13 +136,17 @@ public void commandWithLib() {
     IFile file = createMock(IFile.class);
     IPath filePath = Path.fromOSString("home").append("project").append("Hello.g4");
     IPath outputPath = Path.fromOSString("target").append("generated-sources").append("antlr4");
-    IPath libPath = Path.fromOSString("home").append("project").append("lib");
+//    IPath libPath = Path.fromOSString("home").append("project").append("lib");
```

Comment:
Should this be commented or removed? Would someone using this want to switch back?